electron/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "auctions-electron",
+  "version": "1.0.0",
+  "description": "auctions electron app packaging",
+  "main": "main.js",
+  "scripts": {
+    "start": "electron ."
+  },
```

Comment:
As discussed, please add build command that when executed do all necessary steps from building frontend with correct env vars to outputing `electron/distr` folder with built and ready-to-distribute apps